tests/v2/validation/deleting/delete_init_node_test.go

```diff
@@ -0,0 +1,65 @@
+package deleting
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/nodes"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type DeleteInitNodeTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+}
+
+func (d *DeleteInitNodeTestSuite) TearDownSuite() {
+	d.session.Cleanup()
+}
+
+func (d *DeleteInitNodeTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	d.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(d.T(), err)
+
+	d.client = client
+}
+
+func (d *DeleteInitNodeTestSuite) TestDeleteInitNode() error {
```

Comment:
Let's talk offline about this design. I think we should follow the practice we're doing in the other deleting test/scaling tests where we are just grabbing the cluster ID and then proceeding to the functionality you have.

In other words, I think you should have a `delete_init_node.go` file where everything after `GetClusterIDByName` is referenced; the `TestDeleteInitNode` should call one function that lives in `delete_init_node.go` that will then run every other function you have here.